### PR TITLE
Added Ugly Number C++ Implementation

### PR DESCRIPTION
**Pull request by 13muskanp**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

C++ Implementation of Ugly numbers problem

### Description

A frequent question in coding rounds of many companies

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1286

### CODE REVIEW

File: C-Plus-Plus/math/ugly_numbers.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+{
+	//code
+	int t;
+	cin>>t;
+	
+	long long arr[10000];
+	int c2 = 0, c3 = 0, c5 = 0;
+	arr[0] = 1;
+	
+	for(int i=1; i<10000; i++)
```
Comment (Kajol-Kumari) [inline C-Plus-Plus/math/ugly_numbers.cpp:32]: why are we limiting the n to 10000 here?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]

**Review by 13muskanp — COMMENTED** [review]
Yes

File: C-Plus-Plus/math/ugly_numbers.cpp (hunk 1 above)
Comment (13muskanp) [reply C-Plus-Plus/math/ugly_numbers.cpp:32]: No specific reason, can be scaled

**Review by 13muskanp — COMMENTED** [review]
Yes

**Pull request merged**